doc/modules/motion_correction.rst

```diff
@@ -7,26 +7,26 @@ Motion/drift correction
 Overview
 --------
 
-Mechanical drifts, often observed in recordings, are currently a major issue for spike sorting. This is especially striking
-with the new generation of high-density devices used in-vivo such as the neuropixel electrodes.
-The first sorter that has introduced motion/drift correction as a prepossessing step was kilosort2.5 (see [Steinmetz2021]_)
+Mechanical drift, often observed in recordings, is currently a major issue for spike sorting. This is especially striking
+with the new generation of high-density devices used for in-vivo electrophyisology such as the neuropixel electrodes.
+The first sorter that has introduced motion/drift correction as a prepossessing step was Kilosort2.5 (see [Steinmetz2021]_)
 
 Long story short, the main idea is the same as the one used for non-rigid image registration, for example with calcium
 imaging. However, because with extracellular recording we do not have a proper image to use as a reference, the main idea
 of the algorithm is create an "image" via the activity profile of the cells during a given time window. Assuming this
 activity profile should be kept constant over time, the motion can be estimated, by blocks, along the probe's insertion axis
-(i.e. depth) so that we can interpolate the traces to compensate this estimated motion.
+(i.e. depth) so that we can interpolate the traces to compensate for this estimated motion.
 Users with a need to handle drift were currently forced to stick to the use of Kilosort2.5 or pyKilosort (see [Pachitariu2023]_). Recently, the Paninski
 group from Columbia University introduced a possibly more accurate method to estimate the drift (see [Varol2021]_
-and [Windolf2023]_), but this new method was not properly integrated in any sorter.
+and [Windolf2023]_), but this new method was not properly integrated into any sorter.
```

Comment:
in vs into
Interesting.

Reply:
@ h-mayorquin Yeah super small, but into implies entry like motion. We integrated it into something. Whereas as in is a bit more stationary so. That feature was already in the sorter.